src/convert.jl

```diff
@@ -1026,3 +1026,57 @@ function convert(::Type{HParallelotope}, Z::AbstractZonotope{N}) where {N}
     end
     return HParallelotope(D, c)
 end
+
+"""
+    convert(::Type{Zonotope}, cp::CartesianProduct{N, <:AbstractZonotope{N}
+            , <:AbstractZonotope{N}}) where {N}
+
+Converts a cartesian product of two zonotopes into a zonotope.
+
+### Input
+
+- `Zonotope` -- type used for dispatch
+- `cp`       -- cartesian product of two zonotopes
+
+### Output
+
+A zonotope.
+
+### Notes
+
+This function requires the use of `SparseArrays`.
```

Comment:
`SparseArrays` is a `Base` library that we anyway require, so I would not mention the requirement.
```suggestion
This implementation creates a `Zonotope` with sparse vector and matrix representation.
```